src/stream/src/executor/debug/trace.rs

```diff
@@ -23,7 +23,7 @@ use crate::executor::error::StreamExecutorError;
 use crate::executor::monitor::StreamingMetrics;
 use crate::executor::{ExecutorInfo, Message, MessageStream};
 use crate::task::ActorId;
-
+const ENABLE_EXECUTOR_ROW_COUNT: bool = false;
```

Comment:
Better to add docs to this global variable:

/// Set to true to enable per-executor row count metrics. This will produce a lot of timeseries and might affect the prometheus performance. If you only need actor input and output rows data, see `stream_actor_in_record_cnt` and `stream_actor_out_record_cnt` instead.